<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLighting</class>
 <widget class="QWidget" name="WidgetLighting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>147</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lighting</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="WidgetCheckerSphere" name="checkerSphere">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelShadowDiffuse">
       <property name="text">
        <string>Diffuse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderLightDiffuse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The diffuse component of the light source (r=g=b)</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelShadowAmbient">
       <property name="text">
        <string>Ambient</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderLightAmbient">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The ambient component of the light source (r=g=b)</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelShadowSpecular">
       <property name="text">
        <string>Specular</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderLightSpecular">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The specular component of the light source (r=g=b)</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelShadowShininess">
       <property name="text">
        <string>Shininess</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderLightShininess">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Specular exponent</string>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>64</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelTwoSides">
       <property name="text">
        <string>2-Sides</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QCheckBox" name="checkBoxLightTwoSides">
       <property name="toolTip">
        <string>Shading for both sides</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelHighlighBack">
       <property name="text">
        <string>High back </string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QCheckBox" name="checkBoxLightDistinctBackColor">
       <property name="toolTip">
        <string>Distinct color for the back side</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QToolButton" name="toolButtonLightDistinctBackColor">
       <property name="toolTip">
        <string>The color of the back side</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0" colspan="5">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelSSAO">
       <property name="text">
        <string>SSAO</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="4">
      <widget class="QComboBox" name="comboBoxSSAOAlgorithm">
       <property name="toolTip">
        <string>Smooth pattern for soft shadow</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelSSAORadius">
       <property name="text">
        <string>Radius</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderSSAORadius">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Distance of light source to the scene center (relative to scene radius)</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="labelSSAOIntensity">
       <property name="text">
        <string>Intensity</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderSSAOIntensity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Distance of light source to the scene center (relative to scene radius)</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="labelSSAOBias">
       <property name="text">
        <string>Bias</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderSSAOBias">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Distance of light source to the scene center (relative to scene radius)</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="labelSSAOSharpness">
       <property name="text">
        <string>Sharpness</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderSSAOSharpness">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Distance of light source to the scene center (relative to scene radius)</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>12800</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>4000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="labelEDL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>EDL</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="2">
      <widget class="QCheckBox" name="checkBoxEyeDomeLighting">
       <property name="toolTip">
        <string>Enable/Disable eye dome lighting for point clouds</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="3" colspan="2">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>63</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="labelTransparent">
       <property name="text">
        <string>Transparent</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1" colspan="2">
      <widget class="QCheckBox" name="checkBoxTransparency">
       <property name="toolTip">
        <string>Enable/Disable transparency effect</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="3" colspan="2">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>63</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="15" column="0" colspan="5">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="labelShadow">
       <property name="text">
        <string>Shadow</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QCheckBox" name="checkBoxShadow">
       <property name="toolTip">
        <string>Enable/Disable shadowing</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="16" column="2" colspan="3">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="labelShadowSmoothPattern">
       <property name="text">
        <string>PCF filter</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1" colspan="4">
      <widget class="QComboBox" name="comboBoxShadowSmoothPattern">
       <property name="toolTip">
        <string>Smooth pattern for soft shadow</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="labelShadowLightDistance">
       <property name="text">
        <string>Distance</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderShadowLightDistance">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Distance of light source to the scene center (relative to scene radius)</string>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="labelShadowSoftness">
       <property name="text">
        <string>Softness</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderShadowSoftness">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Width of the penumbra region</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="labelShadowDarkness">
       <property name="text">
        <string>Darkness</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1" colspan="4">
      <widget class="QSlider" name="horizontalSliderShadowDarkness">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The darkness of the umbra region</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetCheckerSphere</class>
   <extends>QOpenGLWidget</extends>
   <header>widget_checker_sphere.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>horizontalSliderLightDiffuse</tabstop>
  <tabstop>horizontalSliderLightAmbient</tabstop>
  <tabstop>horizontalSliderLightSpecular</tabstop>
  <tabstop>horizontalSliderLightShininess</tabstop>
  <tabstop>checkBoxLightTwoSides</tabstop>
  <tabstop>checkBoxLightDistinctBackColor</tabstop>
  <tabstop>toolButtonLightDistinctBackColor</tabstop>
  <tabstop>comboBoxSSAOAlgorithm</tabstop>
  <tabstop>horizontalSliderSSAORadius</tabstop>
  <tabstop>horizontalSliderSSAOIntensity</tabstop>
  <tabstop>horizontalSliderSSAOBias</tabstop>
  <tabstop>horizontalSliderSSAOSharpness</tabstop>
  <tabstop>checkBoxEyeDomeLighting</tabstop>
  <tabstop>checkBoxTransparency</tabstop>
  <tabstop>checkBoxShadow</tabstop>
  <tabstop>comboBoxShadowSmoothPattern</tabstop>
  <tabstop>horizontalSliderShadowLightDistance</tabstop>
  <tabstop>horizontalSliderShadowSoftness</tabstop>
  <tabstop>horizontalSliderShadowDarkness</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
